docker-compose.yml

```
@@ -21,6 +21,8 @@ services:
       service: nginx
     volumes:
       - ./config/nginx.conf:/etc/nginx/nginx.conf
+    depends_on:
+      - inferno
```

You can solve this generically, by adding a [healthcheck](https://docs.docker.com/reference/compose-file/services/#healthcheck) to the inferno service, and then changing this `depends_on` to…

```suggestion
    depends_on:
      inferno: service_healthy
```

---

This is not supported in docker compose version 3. ([documentation](https://docs.docker.com/compose/releases/release-notes/#compose-file-version-21-and-up-4))

At most, what we can do is to use `service_started` which guarantees that at least the container started before starting up `nginx`.

```suggestion
    depends_on:
      inferno:
        condition: service_started
```